sos/report/plugins/networking.py

```diff
@@ -349,6 +349,14 @@ def setup(self):
             "/run/systemd/network"
         ])
 
+        # Netplan only consumes files with the `.yaml` extension (LP#1815734),
+        # so give visibility on other files that might be present.
+        self.add_cmd_output([
+            "ls -la /etc/netplan",
+            "ls -la /lib/netplan",
+            "ls -la /run/netplan",
```

Comment:
There's a new function that was added a couple of months ago, which may be better aligned here, something like the following could work here.

```python
self.add_dir_listing([
    "/etc/netplan",
    "/lib/netplan",
    "/run/netplan",
])
```